osgp/platform/osgp-adapter-ws-shared/src/main/java/org/opensmartgridplatform/adapter/ws/infra/jms/LoggingMessageSender.java

```diff
@@ -26,6 +29,9 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(LoggingMessageSender.class);
 
+  @Value("${application.createJsonMessage:false}")
```

Comment:
Application properties are not camelCase and it would be nice to reflect some meaning/context in the property name so it is obvious that it controls the (audit) logging messages.

Reply:
application properties are lowercase now. It is actually just a setting that results in creating json messages rather than serialized messages. It can be used for other purposes than just the audit logging as well so it seems to me the naming reflects the functionality of it quite well.